src/test/java/uk/gov/beis/els/api/common/ApiValuesFromEnumValidatorTest.java

```diff
@@ -0,0 +1,73 @@
+package uk.gov.beis.els.api.common;
+
+import static org.junit.Assert.assertEquals;
+
+import java.util.Set;
+import javax.validation.ConstraintViolation;
+import javax.validation.Validation;
+import javax.validation.Validator;
+import javax.validation.ValidatorFactory;
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.mockito.junit.MockitoJUnitRunner;
+import uk.gov.beis.els.categories.prorefrigeratedcabinets.model.ClimateClass;
+import uk.gov.beis.els.categories.prorefrigeratedcabinets.model.ProRefrigeratedCabinetsForm;
+import uk.gov.beis.els.model.RatingClass;
+
+@RunWith(MockitoJUnitRunner.class)
+public class ApiValuesFromEnumValidatorTest {
+
+  private Validator validator;
+
+  @Before
+  public void setup() {
+    ValidatorFactory validatorFactory = Validation.buildDefaultValidatorFactory();
+    validator = validatorFactory.getValidator();
+  }
+
+  @Test
+  public void validEnumValue_displayName_assertNoErrors() {
+    String validEnum = ClimateClass.THREE.getDisplayName();
+    ProRefrigeratedCabinetsForm form = getForm();
+    form.setClimateClass(validEnum);
+
+    Set<ConstraintViolation<ProRefrigeratedCabinetsForm>> errors = validator.validate(form);
+
+    assertEquals(0, errors.size());
```

Comment:
Nit; slightly more semantic to use AssertJs `isEmpty` e.g. `assertThat(errors).isEmpty();`
Same with the other tests

Reply:
done